cmd/create/cluster/cmd.go

```diff
@@ -2629,3 +2641,38 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 func getRolePrefix(clusterName string) string {
 	return fmt.Sprintf("%s-%s", clusterName, helper.RandomLabel(4))
 }
+
+func validateManagedAccountRoles(r *rosa.Runtime, prefix string, awsClient aws.Client) error {
```

Comment:
My only though is if these 2 funcs belong in the `create cluster cmd` or maybe can also be used for other commands, and should be placed in some common pkg?

Reply:
Good point.
Reused `ValidateAccountRolesManagedPolicies` from the `roles` package.
Moved `ValidateUnmanagedAccountRoles` to the `roles` package as well, though didn't find yet a place to reuse the function.